.circleci/config.yml

```diff
@@ -155,6 +158,24 @@ jobs:
             export CONF_TEST_PATH=~/protobuf/conformance/conformance-test-runner
             ./conformance/test-conformance.sh js
 
+  conformance_native:
+    executor:
+      name: native
```

Comment:
Not sure what I have to put here. CI pipeline fail with this error:
`Cannot find a definition for executor named native`

Reply:
Thanks for the tip! It's working now 😄